docs/k8s-helm.md

```diff
@@ -91,28 +86,101 @@ export RELEASE=timeplus
 helm -n $NS install -f values.yaml $RELEASE timeplus/timeplus-enterprise
 ```
 
-You can run `kubectl -n $NS get jobs` to check whether `timeplus-provision` is completed or not. Once this job is completed, you can start use Timeplus Enterprise.
+You should see following output within a couple of minutes.
+
+```bash
+NAME: timeplus
+LAST DEPLOYED: Tue Aug  6 13:41:31 2024
+NAMESPACE: timeplus
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+Your Timeplus Enterprise stack is deployed! It may take a few minutes before all the services are up and running.
+You can run `kubectl -n timeplus get jobs` to check whether `timeplus-provision` is completed or not. Once this job is completed, you can start use Timeplus Enterprise.
+```
 
-It will take 1 or 2 minutes to start the whole stack, run following `kubectl get pods -n $NS` to check the stack status, for example:
+To make sure everything is up and running, run `kubectl get pods -n ${NS}` to check the stack status
```

Comment:
let's keep it consistent. Line 86 uses $NS. Probably no need to change this line to ${NS}, because there is no special char, $NS is a bit cleaner.

Reply:
updated